packages/backend/test/peripherals/database/migrations.test.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'earljs'
+import { readdirSync } from 'fs'
+import path from 'path'
+
+describe('migrations', () => {
+  it('migrations have consecutive numbering except for 20', () => {
```

Comment:
don't we have already some other tests for migrations?

Reply:
Yes we do: `packages/backend/test/peripherals/database/shared/Database.test.ts`. Maybe it makes sense to merge these test files?